com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/cf/metadata/util/MtaMetadataUtil.java

```diff
@@ -14,16 +14,18 @@
 import com.sap.cloud.lm.sl.cf.core.cf.metadata.MtaMetadataAnnotations;
 import com.sap.cloud.lm.sl.cf.core.cf.metadata.MtaMetadataLabels;
 
+import io.micrometer.core.instrument.util.StringUtils;
```

Comment:
I think it will be better to use StringUtils from apache commons (org.apache.commons.lang3)